dataverse/orgsvc/C#-NETCore/BulkOperations/UpsertMultiple/Program.cs

```diff
@@ -0,0 +1,327 @@
+using Microsoft.PowerPlatform.Dataverse.Client;
+using Microsoft.Xrm.Sdk;
+using Microsoft.Xrm.Sdk.Messages;
+using Microsoft.Xrm.Sdk.Query;
+using System.Diagnostics;
+using System.Net;
+using System.ServiceModel;
+
+namespace PowerPlatform.Dataverse.CodeSamples
+{
+    class Program
+    {
+        /// <summary>
+        /// Contains the application's configuration settings. 
+        /// </summary>
+        IConfiguration Configuration { get; }
+
+
+        /// <summary>
+        /// Constructor. Loads the application configuration settings from a JSON file.
+        /// </summary>
+        Program()
+        {
+
+            // Get the path to the appsettings file. If the environment variable is set,
+            // use that file path. Otherwise, use the runtime folder's settings file.
+            string? path = Environment.GetEnvironmentVariable("DATAVERSE_APPSETTINGS");
+            path ??= "appsettings.json";
+
+            // Load the app's configuration settings from the JSON file.
+            Configuration = new ConfigurationBuilder()
+                .AddJsonFile(path, optional: false, reloadOnChange: true)
+                .Build();
+        }
+        static void Main()
+        {
+            if (Settings.UseElastic == true)
+            {
+                Console.WriteLine("This sample doesn't currently support elastic tables.");
+                Console.WriteLine("Change UseElastic in Settings.cs to false and try again.");
+                return;
+            }
+
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
+
+            Program app = new();
+
+            int numberOfRecords = Settings.NumberOfRecords; //100 by default
+            string tableSchemaName = "sample_Example";
+            string tableLogicalName = tableSchemaName.ToLower(); //sample_example            
+
+            // Create a Dataverse service client using the default connection string.
+            ServiceClient serviceClient =
+                new(app.Configuration.GetConnectionString("default"))
+                {
+                    UseWebApi = false
+                };
+
+            try
+            {
+                // Create sample_Example table for this sample
+                Utility.CreateExampleTable(
+                    serviceClient: serviceClient,
+                    tableSchemaName: tableSchemaName,
+                    isElastic: Settings.UseElastic,
+                    createAlternateKey: Settings.CreateAlternateKey);
+
+                // Confirm the table supports UpsertMultiple
+                if (Settings.UseElastic && !Utility.IsMessageAvailable(
+                    service: serviceClient,
+                    entityLogicalName: tableLogicalName,
+                    messageName: "UpsertMultiple"))
+                {
+                    Console.WriteLine($"The UpsertMultiple message is not available " +
+                        $"for the {tableSchemaName} table.");
+                }
```

Comment:
This code is not required since you are already returning if UseElastic is true at the start of the Main(). Standard table does not have UpsertMultiple filter for every entity. Since UpsertMultiple is eventually executing CreateMultiple or UpdateMultiple.